{"uid":"6cc3c6a75dad4cf","name":"CT004 REALIZAR SAQUE COM SUCESSO","fullName":"testathon.test.saque.SaqueTest.testDeveFazerSaqueComSucesso","historyId":"17c9d96c1f9ec818345cfd46a99b3092","time":{"start":1739195213763,"stop":1739195256635,"duration":42872},"status":"failed","statusMessage":"expected [Saque realizado com sucesso] but found [Sacar]","statusTrace":"java.lang.AssertionError: expected [Saque realizado com sucesso] but found [Sacar]\n\tat org.testng.Assert.fail(Assert.java:97)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:575)\n\tat org.testng.Assert.assertEquals(Assert.java:585)\n\tat testathon.test.saque.SaqueTest.testDeveFazerSaqueComSucesso(SaqueTest.java:59)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"abrirNavegador","time":{"start":1739195204349,"stop":1739195207230,"duration":2881},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false},{"name":"setUp","time":{"start":1739195207235,"stop":1739195213733,"duration":6498},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"expected [Saque realizado com sucesso] but found [Sacar]","statusTrace":"java.lang.AssertionError: expected [Saque realizado com sucesso] but found [Sacar]\n\tat org.testng.Assert.fail(Assert.java:97)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:575)\n\tat org.testng.Assert.assertEquals(Assert.java:585)\n\tat testathon.test.saque.SaqueTest.testDeveFazerSaqueComSucesso(SaqueTest.java:59)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n","steps":[],"attachments":[{"uid":"5900a6e9af619a65","name":"Screenshot on Failure","source":"5900a6e9af619a65.png","type":"image/png","size":67090}],"parameters":[],"stepsCount":0,"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":1,"attachmentStep":false},"afterStages":[{"name":"fecharNavegador","time":{"start":1739195256644,"stop":1739195257104,"duration":460},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false}],"labels":[{"name":"package","value":"testathon.test.saque.SaqueTest"},{"name":"testClass","value":"testathon.test.saque.SaqueTest"},{"name":"testMethod","value":"testDeveFazerSaqueComSucesso"},{"name":"parentSuite","value":"TesteNG com Selenium Grid"},{"name":"suite","value":"Teste Firefox"},{"name":"subSuite","value":"testathon.test.saque.SaqueTest"},{"name":"host","value":"MacBook-Air-de-Renato.local"},{"name":"thread","value":"56871@MacBook-Air-de-Renato.local.TestNG-test=Teste Firefox-1(18)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"feature","value":"Testes de Saque"},{"name":"severity","value":"critical"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"browser","value":"firefox"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"6cc3c6a75dad4cf.json","parameterValues":["firefox"]}